### PR TITLE
Adding support for ILiteralOperation to control flow graph

### PR DESCRIPTION
**Pull request by chborl**

### CODE REVIEW

**Comment by chborl** [comment]
Tagging @AlekseyTs , @mavasani , @333fred  for review.

File: src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs [hunk 1]
```diff
@@ -4004,6 +4004,11 @@ public override IOperation VisitTypeOf(ITypeOfOperation operation, int? captureI
             return new TypeOfExpression(operation.TypeOperand, semanticModel: null, operation.Syntax, operation.Type, operation.ConstantValue, operation.IsImplicit);
         }
 
+        public override IOperation VisitLiteral(ILiteralOperation operation, int? captureIdForResult)
```
Comment (333fred) [inline src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs]: This needs to move all the way out of the region.

File: src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_ILiteralOperation.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
```
Comment (jaredpar) [inline src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_ILiteralOperation.cs]: Double blank line above.

File: src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_ILiteralOperation.cs [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+{
+    public partial class IOperationTests : SemanticModelTestBase
+    {
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)]
+        [Fact]
+        public void LiteralFLow_01()
```
Comment (jaredpar) [inline src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_ILiteralOperation.cs]: >LiteralFLow_01 [](start = 20, length = 14)

Should we have a test for at least one other literal type here: say string? Don't think we need to go through the full set here.

Comment (jaredpar) [reply src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_ILiteralOperation.cs]: Ah just caught up on email which said this is sufficient. 

---
In reply to: [186608888](https://github.com/dotnet/roslyn/pull/26691#discussion_r186608888) [](ancestors = 186608888)

**Pull request merged**